homeassistant/components/deconz/__init__.py

```diff
@@ -119,6 +119,13 @@ def async_add_remote(sensors):
 
     deconz.start()
 
+    device_registry = await \
+        hass.helpers.device_registry.async_get_registry()
+    device_registry.async_get_or_create(
+        [['bridgeid', deconz.config.bridgeid]], 'Dresden Elektronik',
+        deconz.config.modelid, [['Ethernet', deconz.config.mac]],
```

Comment:
WiFi and Ethernet share the same mac address space, so we should also group them into the same connection category.

Reply:
So network?